digits/dataset/images/classification/views.py

```diff
@@ -167,6 +203,37 @@ def from_files(job, form):
     if not image_folder:
         image_folder = None
 
+    augmentation = {
+    }
+
+    if form.has_augmentation_contrast.data:
+        augmentation['contrast'] = {
+            'strength': form.augmentation_contrast_strength.data,
+            'probability': form.augmentation_contrast_probability.data
+        }
+
+    if form.has_augmentation_hue.data and job.image_dims[2] == 3:
+        augmentation['hue'] = {
+            'angle': form.augmentation_hue_angle.data,
```

Comment:
do you mean to use `hue_angle_min` and `hue_angle_max`? The field `hue_angle` doesn't seem to be rendered in the HTML template...